1166 detail short url

dstopp opened the pull request:
- Fixes #1166
- Were any Django, import pipeline, table_schema, or dictionary files modified? N
  - If YES:
    - Database used: opus3_test_210526
    - All Django tests pass: NA
    - FLAKE8 run on modified code: NA
- Were any JavaScript or CSS files modified? Y
  - If YES:
    - JSHINT run on all affected files: Y/NA
    - Tested on Chrome, Firefox, Safari, Edge, and iOS: Y/NA
      (Remember to test all browser sizes)
    - Tested for race conditions (with delayed API calls if applicable): Y/NA

Description of changes:

Known problems:

dstopp committed:
Initial changes to allow user to copy a 'short' url for detail

-add icon 'share'
-create href that contains only pertinent detail information

dstopp committed:
Minor changes

-Update tooltip
-Make the URL copied message symmetric
-Make detail cart/share icons a bit larger for readability

dstopp committed:
Minor change for safari spacing

dstopp committed:
Fix error message to go to screen instead of console

juzen2003 reviewed (changes requested):
Issues:
- The perfect scrollbar on the detail page is not scrolling, it will show up at the beginning but will be gone once we start to scroll.
- When share button is clicked, the URL will change to the short one and if we refresh the page, the URL will go back to the long one (but we still stay in the detailed tab), not quite sure if this is what we expect.
- Similar to previous one, this exists in previous pull, not related to the changes here. When the cart button is clicked in detailed page, URL will change and if we refresh the page, the page will reload to the search tab.

dstopp committed:
Address pull request comments

-fix perfect scrollbar
-when share button or trash/add button clicked, do not change url

juzen2003 commented on opus/application/static_media/js/detail.js:
```diff
@@ -37,8 +37,16 @@ var o_detail = {
+                        let urlToShare = $(this).attr("href");
+                        o_detail.copyToClipboard(urlToShare)
+                            .then(() => o_detail.showShareMessage("URL copied!"))
+                            .catch(() => o_detail.showShareMessage("Error copying URL"))
```
miss `;` at the end

juzen2003 reviewed (changes requested):
- Database used: opus3_test_210526
- All Django tests pass: Y
- JSHINT run on all affected files: Y
- FLAKE8 run on all affected files: Y
- Tested on Chrome, Firefox, Safari, and iOS: Y
  (Remember to test all browser sizes)
- Tested for race conditions (with delayed API calls if applicable): Y

Only one minor cleanup needed, please add ";" at the end of line 45 in detail.js

dstopp committed:
added missing ;

dstopp committed:
Update to tooltip
